content/guides/trash/permanently-delete-a-file.md

```diff
@@ -0,0 +1,19 @@
+---
+rank: 2
+related_endpoints:
+ - delete_files_id
+related_guides: []
+required_guides: []
+related_resources:
+  - files
+---
+
+# Permanently delete a file
+
+Once a file has been moved to the trash, it will remain in the trash for 30
+days, by default, before being purged. Administrators of Business or
+Enterprise accounts can alter the purge window. If you wish to permanently
```

Comment:
Style note: Ive been trying to keep the use of `you` to a limit. Often you can just get away without it or to use `application` instead. In this case you could say "To permanently delete..." or "If an application wants to permanently"...